### ISSUE TITLE
Feature Ui music player: Add a "Loop" toggle button 

### ISSUE THREAD

**Issue by Gero5** [mpogue2/SquareDesk#1211]
Except for the braces in the music envelope shape there is no indication wether loop is turned of or off. there is also no possibility to toggle loop from the UI other than pressing "L" on the keyboard.

**Feature Request:**
Add a button called "Loop" or "Loop On/Off" below the music envelope. The button is highlighted when Loop is On. When clicked it toggles the Loop feature for ON to OFF and vice versa.

See also #1210 #1209

**Comment by mpogue2** [comment]
I like this idea.  I'm thinking, though, that it should be a loop **icon** (as some of the other music players use),
and it should be integrated with other stateful controls.

For example, here's Mixxx (SNAP and LOOP shown here are stateful controls):

<img width="639" alt="image" src="https://github.com/user-attachments/assets/ef62149f-1ef8-4e8d-84a9-046044719e42">

Note: the Mixxx count there is not the "loop counter" you suggested, it's a "loop length", so here is where we would diverge from the Mixxx model.

**Comment by Gero5** [comment]
yes. An icon knob or indicator is nice.

> and it should be integrated with other stateful controls.

What are you thinking of? Normalize / Autostart / Boost?

**Comment by mpogue2** [comment]
There are more Mixxx controls that I didn't explore before.  Here's some more info on those:

![MixxxExampleControls](https://github.com/user-attachments/assets/9d50f01d-0fd9-45b8-a585-40e801401bdc)

I notice a couple of things:
- icons are the ones you pointed out for start/end loop
- entire loop is highlighted in transparent green (rather than the brackets we use now)
- You can define intro and outro regions using the 4 buttons shown.  Right now, SquareDesk internally treats Intro/Outro (singers) and Start/End Loop (patter) as the same numbers.  Can't have both.  In the above pic, we can see both in the same song.  I'm not sure why we'd want both, just thought I'd point that out.
- The loop control icon to the right is Loop **Entire Song** Enable/Disable, not Enable/Disable Loop (the green overlay).

**Comment by mpogue2** [comment]
Yes, I'm thinking other state icons could be for Normalize, Loop Enable/Disable, Boost, etc... Over to the right seems like a logical place to put those.

**Comment by mpogue2** [comment]
Shipped, therefore closing.
